code/game/objects/items/devices/radio/radio_objects.dm

```diff
@@ -541,9 +549,15 @@ GLOBAL_LIST_EMPTY(deadsay_radio_systems)
 
 	return null
 
+/obj/item/radio/proc/show_examine_hotkeys()
+	. = list()
+	. += "<span class='notice'><b>Alt-Click</b> to toggle [src]'s hotmic!</span>"
+	. += "<span class='notice'><b>Ctrl-Shift-Click</b> to toggle [src]'s speaker!</span>"
+
 /obj/item/radio/examine(mob/user)
 	. = ..()
-	. += "<span class='notice'><b>Alt-Click</b> to toggle [src]'s hotmic!</span>"
+	. += show_examine_hotkeys()
+
```

Comment:
```suggestion
	. += "<span class='notice'>You can transmit messages from [src] without the hotmic by using :l or :r whilst holding it in your left or right hand.</span>"
	. += show_examine_hotkeys()

```
This mechanic should also be pointed out for people.

Reply:
maybe put that in the examine more, it is already getting ridiculously long